imap/quota_db.c

```diff
@@ -409,13 +419,27 @@ EXPORTED int quota_foreach(const char *prefix, quotaproc_t *proc,
     foreach_d.proc = proc;
     foreach_d.rock = rock;
     foreach_d.tid = tid;
+    foreach_d.use_conv = use_conv;
 
     r = cyrusdb_foreach(qdb, search, strlen(search), NULL,
                      do_onequota, &foreach_d, tid);
 
     return r;
 }
 
+EXPORTED int quota_foreach(const char *prefix, quotaproc_t *proc,
+                           void *rock, struct txn **tid)
+{
+    return _quota_foreach(prefix, proc, rock, tid, 0);
+}
+
+EXPORTED int quota_foreach_withconversations(const char *prefix,
```

Comment:
nit: I would have added a flag to `quota_foreach` rather than a new function here, but we have no hard rules for either. Is there a chance that we'll want to extend `quota_foreach` in the future with additional options? Did you weigh both variants against each before you chose a new function, if so what was the rationale?

Reply:
Done